docs/rules/no-pending-tests.md

```diff
@@ -0,0 +1,41 @@
+# Disallow Pending Tests (no-pending-tests)
+
+Mocha allows specification of pending tests, which represent tests that aren't yet implemented, but are intended to be implemented eventually. These are designated like a normal mocha test or suite, but with only the first argument provided (no callback for the actual implementation). For example: `describe('unimplemented bdd suite');`
```

Comment:
AFAIK mocha currently doesn’t support pending suites. If the second argument of `describe` is missing an error will be thrown.
There is an [upstream issue](https://github.com/mochajs/mocha/issues/1744) about this issue, but as I understand it, it hasn’t been decided if pending suites will be implemented or if the error message will be improved.